### DISCUSSION TITLE
Helm update possibilities.

### DISCUSSION THREAD

**Discussion by DaanSelen** [nextcloud/all-in-one#4816] [Ideas] [answered]
### Is your feature request related to a problem? Please describe. 
The problem is that the helm chart should be able to reproduce the Nextcloud AIO features such as back, but why does it not?

### Describe the solution you'd like
A better use or integration of the helm chart with the nextcloud-aio applications.

### Describe alternatives you've considered
Deploying all applications separately with helm, but then there is no use in using the AIO chart.

### Additional context
I'd like to be able to customize the deployment helm makes with the chart in this repository. The documentation right now is a bit minimal.

Thanks.

**Comment by szaimen** [comment, marked as answer]
Hi, I dont understand what you are requesting. It looks like multiple things that are unrelated to each other. Can you please paraphrase so that I am able to better understand what you want? Thanks!

**Reply by DaanSelen** [reply]
Hello Szaimen, I read in the helm chart readme that quite a couple features get discarded when using the chart, why is this?

**Reply by szaimen** [reply]
The main reason is that most of the features are built into the [mastercontainer](https://github.com/nextcloud/all-in-one/tree/main/Containers/mastercontainer) and its web interface which is built around docker and uses the docker api internally which is not possible to be used on kubernetes.

**Reply by DaanSelen** [reply]
Are there no alternatives?

**Reply by szaimen** [reply]
No not really

**Reply by DaanSelen** [reply]
And from what I tested, can I really tell the AIO with the helm chart to not deploy certain contains? Such as talk?

**Reply by szaimen** [reply]
you should be able to deploy talk with the helm chart. 
if you look at the following variables, these should allow you to enable additional features next to normal Nextcloud:
https://github.com/nextcloud/all-in-one/blob/cbe0a89676001bff86cf2ee9c7aea0767ce21b0d/nextcloud-aio-helm-chart/values.yaml#L14-L20

**Reply by DaanSelen** [reply]
From my experience this only disabled them, but still deployed them. I could be wrong... doing some re-testing soon, I'll update this discussion once I know more! Thanks already.

**Reply by szaimen** [reply]
hm, it shouldnt if you e.g. look at https://github.com/nextcloud/all-in-one/blob/cbe0a89676001bff86cf2ee9c7aea0767ce21b0d/nextcloud-aio-helm-chart/templates/nextcloud-aio-clamav-deployment.yaml#L1
